sqlite3/src/lib.rs

```diff
@@ -1646,3 +1674,151 @@ fn sqlite3_safety_check_sick_or_ok(db: &sqlite3Inner) -> bool {
         }
     }
 }
+
+#[no_mangle]
+pub unsafe extern "C" fn sqlite3_table_column_metadata(
```

Comment:
Good to add reference to the SQLite docs in the docstring here: https://sqlite.org/c3ref/table_column_metadata.html